sigmastate/src/test/scala/special/sigma/SigmaDslSpecification.scala

```diff
@@ -7335,6 +7335,122 @@ class SigmaDslSpecification extends SigmaDslTesting
         )))
   }
 
+  /* Actual test fails for v4.x with
+   * `new StagingException("Don't know how to evaluate(s1252 -> Placeholder(BaseElemLiftable<Int>))")`
+   * but that cannot be explicitly stated in cases. Therefore we use different exception (None.get)
+   * and also `allowDifferentErrors` flag.
+   * Test passes for v5.x so we use flag `allowNewToSucceed`.
+   */
+  property("Random headers access and comparison (originaly from spam tests)") {
+    val (_, _, _, ctx, _, _) = contextData()
+
+    verifyCases(
+      Seq(
+        ctx -> Expected(Failure(new NoSuchElementException("None.get")), 0)
```

Comment:
Please specify full expectations for v5.0 like in this property https://github.com/ScorexFoundation/sigmastate-interpreter/blob/c5e005f37bdbfc4693e9a51a08121b21a1d987e3/sigmastate/src/test/scala/special/sigma/SigmaDslSpecification.scala#L6414